libsemigroups_cppyy/todd_coxeter.py

```diff
@@ -95,21 +109,13 @@ def int_to_strategy(self, n):
         else:
             assert False
 
-    def fp_repr_method(self):  # from froidure_pin.py
-        try:
-            element_type_str = "<%s>" % type(self).element_type.short_name
-        except AttributeError:
-            element_type_str = ""
-        plural = "s" if self.nr_generators() > 1 else ""
-        return "<FroidurePin{0} object with {1} generator{2} at {3}>".format(
-            element_type_str, self.nr_generators(), plural, hex(id(self))
-        )
-
     detail.wrap_overload_params_and_unwrap_return_value(
         tc_type, tc_type.strategy, wrap_strategy, int_to_strategy
     )
 
-    detail.wrap_overload_params(tc_type, tc_type.standardize, wrap_standardize)
+    detail.wrap_overload_params_and_unwrap_return_value(
+        tc_type, tc_type.standardize, wrap_standardize, lambda self, x: x
+    )
```

Comment:
This looks a little bit funny, but this is because using the `detail.wrap_overload_params` as opposed to `detail.wrap_overload_params_and_unwrap_return_value` gave me a return value looking like `(<ToddCoxeter object ... >, None)`. Changing the `__do_nothing` function in `detail.py` from returning `args` to returning `None` fixes this, but causes problems in places that I don't understand (the tests failed), and so I've just gone with this approach for now.